proto/minder/v1/minder.proto

```diff
@@ -1887,27 +1887,64 @@ message ListEvaluationResultsResponse {
 message RestType {
     message Fallback {
         int32 http_code = 1 [(buf.validate.field).int32 = { gte: 100, lte: 599 }];
-        string body = 2;
+        string body = 2 [
+            (buf.validate.field).string = {
+                pattern: "^\\s*\\{.*\\}\\s*$",
```

Comment:
I'm not sure what this is trying to match.  This may be a case where we can't qualify the contents or need to use CEL to do so.

Reply:
Yes, it's supposed to be JSON there. so I added the following `^\\s*\\{[\\s\\S]*\\}\\s*$` which should check for opening/closing bracket and match any character including new lines